packages/core/src/common/console-logger-server.ts

```diff
@@ -7,25 +7,42 @@
 
 import { injectable } from "inversify";
 import { LogLevel } from './logger';
-import { ILoggerServer, ILoggerClient } from './logger-protocol';
+import { ILoggerServer } from './logger-protocol';
+
+const originalConsoleLog = console.log;
```

Comment:
Can't we remove the exchange of console.log (etc.) on the backend as well?

Reply:
We cannot. Currently, `console.log` redirects to `ConsoleLoggerServer#log` and that would call `console.log` which would call `ConsoleLoggerServer#log`. You can see, it is already a stack overflow.